src/DynamoUtilities/Md2Html.cs

```diff
@@ -155,20 +180,32 @@ private static string GetToolPath ()
         /// </summary>
         private void KillProcess()
         {
-            if (started && !process.HasExited)
+            if (started)
             {
-                process.Kill();
+                if (!process.HasExited)
+                {
+                    process.Kill();
+                }
                 started = false;
             }
         }
 
         /// <summary>
-        /// Error message
+        /// Can't start Error message
         /// </summary>
         /// <returns>Returns error message</returns>
-        private string GetErrorMessage()
+        private string GetCantStartErrorMessage()
         {
             return @"<p>Can't start '" + GetToolPath() + @"'</p>";
         }
+
+        /// <summary>
+        /// Can't communicate Error message
+        /// </summary>
+        /// <returns>Returns error message</returns>
+        private string GetCantCommunicateErrorMessage()
+        {
+            return @"<p>Can't communicate with '" + GetToolPath() + @"'</p>";
```

Comment:
last thing - is this intended for us or for users? If it's for users then we should put it in a resx file which the localization team can localize.

Reply:
They will show up in the document browser window.

Reply:
I will work adding resources for these strings.